config/prod/prow/config_knative.yaml

```diff
@@ -663,6 +663,10 @@ periodics:
         memory: 12Gi
       limits:
         memory: 16Gi
+  knative-sandbox/eventing-github:
```

Comment:
Do you also need presubmit tests for this repo? If yes, you'll also need to configure under the `presubmits` block.

Reply:
I think we don't need to run the presubmit tests yet since this repo is configured to run GitHub actions and there is no e2e tests yet. 

Let me know if I'm missing something, thanks!